code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -1,4 +1,4 @@
-/obj/item/device/chameleon
+/obj/item/chameleon
 	name = "chameleon-projector"
 	icon_state = "shield0"
 	flags_1 = CONDUCT_1 | NOBLUDGEON_1
```

Comment:
You didn't restore icon for things using the default device.dmi

Reply:
I could just move everything in `device.dmi` to `items_and_weapons` as a catch all; finding all instance of defaulting to `device.dmi` will be extremely difficult.

Still, lemme know what you want me to do.